sound/soc/sof/Makefile

```diff
@@ -7,6 +7,8 @@ snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
 snd-sof-$(CONFIG_SND_SOC_SOF_CLIENT) += sof-client.o
 snd-sof-$(CONFIG_SND_SOC_SOF_DEBUG_PROBES) += probe.o compress.o
 
+snd-sof-$(CONFIG_SND_SOC_SOF_DEBUG_IPC_FLOOD_TEST) += sof-ipc-test.o
```

Comment:
you were so close but you have a client driver driver in the commit message...

Reply:
fixed now. Looks like CI is stalled anyway. So Monday morning might be a good time to check the status